internal/telemetry/scarf.go

```diff
@@ -0,0 +1,110 @@
+package telemetry
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+)
+
+const (
+	base   = "https://home.runme.dev/"
+	client = "Kernel"
+)
+
+type LookupEnv func(key string) (string, bool)
+
+// Returns true if telemetry reporting is enabled, false otherwise.
+func ReportUnlessNoTracking(logger *zap.Logger) bool {
+	if v := os.Getenv("DO_NOT_TRACK"); v != "" && v != "0" && v != "false" {
```

Comment:
I really like `strconv.ParseBool`'s semantic which takes into account sane values and converts them to boolean.

Reply:
Good point 👌.